File: algoritmo/Tarefas.py
from Base_Escalonador1 import TarefaCAV

def criar_tarefas():
    tarefas = [
        TarefaCAV("Deteccao de Obstaculo", 6, prioridade=5, deadline=15, tempo_chegada=7),
        TarefaCAV("Planejamento de Rota", 7, prioridade=1, deadline=9, tempo_chegada=3),
        TarefaCAV("Manutencao de Velocidade", 1, prioridade=7, deadline=40, tempo_chegada=10),
        TarefaCAV("Comunicando com Infraestrutura", 3, prioridade=3, deadline=50, tempo_chegada=11)
    ]
    tarefas.sort(key=lambda tarefa: tarefa.tempo_chegada)  #ordena de acordo com o tempo de chegada
    return tarefas

File: site/algoritmo-js/Tarefas.js
function criar_tarefas() {
    let tarefas = [
        new TarefaCAV("Deteccao de Obstaculo", 6, 7, 5, 15),
        new TarefaCAV("Planejamento de Rota", 7, 3, 1, 9),
        new TarefaCAV("Manutencao de Velocidade", 1, 10, 7, 40),
        new TarefaCAV("Comunicando com Infraestrutura", 3, 11, 3, 50)
    ];
    tarefas.sort((a, b) => a.tempo_chegada - b.tempo_chegada);
    return tarefas;
}
